packages/cli/src/commands/account/register.test.ts

```diff
@@ -1,20 +1,24 @@
-import { testWithGanache } from '@celo/dev-utils/lib/ganache-test'
+import { testWithAnvil } from '@celo/dev-utils/lib/anvil-test'
 import Web3 from 'web3'
-import { testLocally } from '../../test-utils/cliUtils'
+import { testLocallyWithWeb3Node } from '../../test-utils/cliUtils'
 import Register from './register'
 
 process.env.NO_SYNCCHECK = 'true'
 
-testWithGanache('account:register cmd', (web3: Web3) => {
+testWithAnvil('account:register cmd', (web3: Web3) => {
   test('can register account', async () => {
     const accounts = await web3.eth.getAccounts()
 
-    await testLocally(Register, ['--from', accounts[0], '--name', 'Chapulin Colorado'])
+    await testLocallyWithWeb3Node(
```

Comment:
Nit (not Anvil related): 
If I understand correctly, this test succeeds if the command doesn't throw an error. 

Might be more precise to assert the name has been set correctly with an `expect` statement. 

For example, this command [`celocli account:show ARG1`](https://github.com/celo-org/developer-tooling/blob/master/packages/cli/src/commands/account/show.ts) can be used to fetch the account info. I assume there exists a helper function to fetch a contract name. 

https://github.com/celo-org/developer-tooling/blob/3aa75c741b7d29e5da829bb61ed87b9aa6e3bd27/packages/cli/src/commands/account/show.ts#L7-L8

Reply:
agreed, added a meaningful assertion below